settings/DevHome.Settings/Views/FeedbackPage.xaml.cs

```diff
@@ -1,275 +1,302 @@
-        return dotNetVersion.ToString();
-    }
-
-    private string GetOSVersion()
-    {
-        var attrNames = new List<string> { "OSVersionFull" };
-        var attrData = AnalyticsInfo.GetSystemPropertiesAsync(attrNames).AsTask().GetAwaiter().GetResult();
-        var osVersion = string.Empty;
-        if (attrData.ContainsKey("OSVersionFull"))
-        {
-            osVersion = attrData["OSVersionFull"];
-        }
-
-        return osVersion.ToString();
-    }
-
-    private string GetPhysicalMemory()
-    {
-        CultureInfo cultures = new CultureInfo("en-US");
-
-        MEMORYSTATUSEX memStatus = default(MEMORYSTATUSEX);
-        memStatus.dwLength = (uint)Marshal.SizeOf(typeof(MEMORYSTATUSEX));
-        PInvoke.GlobalMemoryStatusEx(out memStatus);
-
-        var availMemKbToGb = Math.Round(memStatus.ullAvailPhys / ByteSizeGB, 2);
-        var totalMemKbToGb = Math.Round(memStatus.ullTotalPhys / ByteSizeGB, 2);
-
-        return "Physical Memory: " + totalMemKbToGb.ToString(cultures) + "GB (" + availMemKbToGb.ToString(cultures) + "GB free)";
-    }
-
-    private string GetProcessorArchitecture()
-    {
-        SYSTEM_INFO sysInfo;
-        PInvoke.GetSystemInfo(out sysInfo);
-        return "Processor Architecture: " + DetermineArchitecture((int)sysInfo.Anonymous.Anonymous.wProcessorArchitecture);
-    }
-
-    private string DetermineArchitecture(int value)
-    {
-        var arch = "Unknown architecture";
-        switch (value)
-        {
-            case 9:
-                arch = "x64";
-                break;
-            case 5:
-                arch = "ARM";
-                break;
-            case 12:
-                arch = "ARMx64";
-                break;
-            case 6:
-                arch = "Intel Itanium-based";
-                break;
-            case 0:
-                arch = "x86";
-                break;
-        }
-
-        return arch;
-    }
-
-    private string GetPlugins()
-    {
-        var pluginService = Application.Current.GetService<IPluginService>();
-        var plugins = pluginService.GetInstalledPluginsAsync(true).Result;
-        var pluginsStr = "Extensions: \n";
-        foreach (var plugin in plugins)
-        {
-            pluginsStr += plugin.PackageFullName + "\n";
-        }
-
-        return pluginsStr;
-    }
-
-    private async void BuildExtensionButtonClicked(object sender, RoutedEventArgs e)
-    {
-        await Launcher.LaunchUriAsync(new ("https://go.microsoft.com/fwlink/?linkid=2234795"));
-    }
-
-    private async void ReportSecurityButtonClicked(object sender, RoutedEventArgs e)
-    {
-        await Launcher.LaunchUriAsync(new ("https://github.com/microsoft/devhome/security/policy"));
-    }
-}
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Linq;
+using System.Management;
```

Comment:
System.Management is getting removed with [PR#986](https://github.com/microsoft/devhome/pull/986).  Please adjust here to match.

Reply:
Replaced with Microsoft.Management.Infrastructure as shown in #986